src/index.js

```diff
@@ -51,10 +51,11 @@ function getModels() {
 function requireAllModels(modelsDir) {
   if (modelsDir) {
     try {
+      const isJSFilename = (fileName) => fileName.endsWith('.js') || (fileName.endsWith('.ts') && !fileName.endsWith('.d.ts'));
+      const isTestFilename = (fileName) => fileName.match(/__tests__|.test.|.spec./g);
       requireAll({
         dirname: modelsDir,
```

Comment:
Exclude all directories that contains `__tests__`:

```suggestion
         dirname: modelsDir,
         excludeDirs: /^__tests__$/,
```

See: https://github.com/felixge/node-require-all#usage